graphql-schemas/schema.yml

```diff
@@ -3663,9 +3663,25 @@ confs:
   - { name: labels, type: json }
   - { name: name, type: string, isRequired: true }
   - { name: app, type: App_v1, isRequired: true }
-  - { name: namespaces, type: Namespace_v1, isList: true, isRequired: true }
+  - { name: prometheusAccess, type: SLOPrometheusAccess_v1, isList: True, isRequired: true }
   - { name: slos, type: SLODocumentSLO_v1, isList: true }
 
+- name: SLOPrometheusAccess_v1
+  datafile: /app-sre/slo-document-1.yml
```

Comment:
the datafile declaration should be used only on GQL types that are top-level to that datafile.